yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -717,22 +766,31 @@ func (yb *TargetYugabyteDB) ExecuteBatch(migrationUUID uuid.UUID, batch EventBat
 			return false, fmt.Errorf("error closing batch: %v", err)
 		}
 
-		updateVsnQuery := batch.GetQueryToUpdateLastAppliedVSN(migrationUUID)
+		updateVsnQuery := batch.GetQueryToUpdateLastAppliedVSN(migrationUUID, numInserts, numUpdates, numDeletes)
```

Comment:
Rename the function to `GetChannelMetadataUpdateQuery()`.
No need to pass the `numInserts`, `numUpdates`, and `numDeletes` from the outside. The function can internally figure out the counts.